README.rst

```diff
@@ -27,6 +27,9 @@ SimPEG
 .. image:: https://zenodo.org/badge/DOI/10.5281/zenodo.1162997.svg
    :target: https://doi.org/10.5281/zenodo.1162997
 
+.. image:: https://img.shields.io/badge/Slack-simpeg-4B0082.svg?logo=slack
+    :target: https://simpeg.slack.com
```

Comment:
this should probably point to "http://slack.simpeg.xyz" (the invite page)

Reply:
Right! Will do.